File: 75-sort-colors/75-sort-colors.java
class Solution {
    public void sortColors(int[] nums) {
        int n=nums.length;
        for(int i=0;i<n;i++)
        {
            
            int minpos=i;
            for(int j=i;j<n;j++)
            {
                if(nums[j]<nums[minpos])
                    minpos=j;
            }
            int temp=nums[i];
            nums[i]=nums[minpos];
            nums[minpos]=temp;
        }
    }
}

File: 75-sort-colors/75-sort-colors.cpp
class Solution {
public:
    void sortColors(vector<int>& nums) {
        int n=nums.size();
        for(int i=1;i<n;i++)
        {
           int pos=i;
            while(pos>0 && nums[pos]<nums[pos-1])
            { swap(nums[pos],nums[pos-1]);
              pos=pos-1;
            }
                      
        }
        //sort(nums.begin(),nums.end());
    }
};